src/server/getters/get-reporting-fees.ts

```diff
@@ -246,11 +248,12 @@ function getMarketsReportingParticipants(db: Knex, reporter: Address, universe:
 }
 
 function getStakedRepResults(db: Knex, reporter: Address, universe: Address, callback: (err: Error|null, result?: { fees: RepStakeResults }) => void) {
-  const crowdsourcerQuery = db.select(["fee_windows.feeWindow", "stakedRep.balance as amountStaked", "payouts.winning", "markets.forking", "markets.marketId"]).from("fee_windows");
+  const crowdsourcerQuery = db.select(["fee_windows.feeWindow", "stakedRep.balance as amountStaked", "payouts.winning", "crowdsourcers.disavowed", "crowdsourcers.completed", "markets.forking", "markets.marketId"]).from("fee_windows");
   crowdsourcerQuery.join("crowdsourcers", "crowdsourcers.feeWindow", "fee_windows.feeWindow");
   crowdsourcerQuery.join("payouts", "crowdsourcers.payoutId", "payouts.payoutId");
   crowdsourcerQuery.join("markets", "markets.marketId", "crowdsourcers.marketId");
-  crowdsourcerQuery.where(db.raw("(payouts.winning or markets.forking)"));
+  crowdsourcerQuery.join("market_state", "markets.marketStateId", "market_state.marketStateId");
```

Comment:
`winning` can serve this purpose. it is null when not finalized, once finalized, all payouts are 0 or 1

Reply:
We actually want to get AWAITING_FINALIZATION though too.

Reply:
As mentioned in chat, winning is available on AWAITING_FINALIZATION for this reason